worker.py

```diff
@@ -56,152 +50,64 @@
 
 # --- Setup logger
 logging.basicConfig(level=FLAGS.logging_level)
+logger = logging.getLogger(__name__)
 
 # --- Configures random generator
 if FLAGS.seed:
   rnd = np.random.RandomState(FLAGS.seed)
 else:
   rnd = np.random.RandomState()
 
+scene = Scene(frame_start=FLAGS.frame_start,
+              frame_end=FLAGS.frame_end,
+              frame_rate=FLAGS.frame_rate,
+              step_rate=FLAGS.step_rate,
+              resolution=(FLAGS.resolution, FLAGS.resolution))
+
+
 # --- Download a few models locally
-asset_source = AssetSource(uri=FLAGS.assets)
+asset_source = KLEVR(uri=FLAGS.assets)
+simulator = Simulator(scene)
 
 # --- Scene static geometry
-floor = asset_source.create({'id': 'Floor', 'static': True, 'position': (0, 0, -0.2)})
+floor, lights, camera = asset_source.get_scene()
```

Comment:
Yeah, I am not too happy with this either. But my thinking was that having three lines that you will want to call together in most cases could be annoying. 
Your example doesn't quite work, since we need to add the floor to both the simulator and the renderer.
So the alternative is something like:
``` python
floor = asset_source.get_scene_geometry()
lights = asset_source.get_lights()
camera = asset_scource.get_camera()
# and then the usual simulator.add(..) calls etc.
```

Reply:
Like this would work?
```
simulator.add(source.get("floor"))
renderer.add(source.get("floor"))
```